build.sbt

```diff
@@ -10,6 +10,7 @@ lazy val `akka-management-root` = project
     `akka-discovery`,
     `akka-discovery-dns`,
     `akka-discovery-kubernetes-api`,
+    `akka-discovery-aws-ec2`,
```

Comment:
nice

Reply:
Would it perhaps be best to name the sub-module `discovery-aws` give there are likely to be other implementations that would likely share much of the same config and dependencies?

Reply:
Will they? All the EC2 ones will you mean ;-)
The container engine ones etc would use kubernetes things, wouldn't they?

Reply:
I'd expect the existing kube api stuff to work against EKS and live there. 

Amazon specific stuff will all need the AWS API for java and it's transitive dependencies. For instance, ECS, route53 or load balancer integrated implementations.

Reply:
Yeah, but that's most likely for EC2 apps anyway? Or would one do those on EKS as well?

Reply:
Perhaps a compromise would be to call it `akka-discovery-aws-api`? Make it clear this is stuff that uses the api but doesn't necessarily preclude other implementations on aws that don't use the aws api for java?

Reply:
I mention this because I have the beginnings of an ECS implementation and will need to know where to put it!

Reply:
I see, the `aws-api` proposal is nice indeed! Let's go with that :)

Reply:
@easel @ktoso akka-discovery-aws-api sounds good, renaming right now :-) cheers!

Reply:
@ktoso @easel 
For the Java package name are we ok with:
`akka.discovery.aws_api` as the main name space ? Java package names can't have "-" but can have "_" :-) 

Then, EC2-based methods can be under `akka.discovery.aws_api.ec2` whereas methods based on something like ECS can be under `akka.discovery.aws_api.ecs` ?? 

@easel What do you think ? 
Cheers!

Reply:
That'd be a bit weird, just `awsapi` :)